### ISSUE TITLE
SPA Quiz Issues/Improvements

### ISSUE THREAD

**Issue by davidcollini** [techlore/website#47]
- Different threat models may work better with using biometrics, while other threat models may be better off not using them

- Explain this to any quiz taker and allow them to decide using threat modeling

**Comment by henry-fisher** [comment]
Hey @IronMaltese — I'm going to merge your other issues with the quiz into this single issue to keep things better organized. 

Your issue [48](https://github.com/techlore/website/issues/48):

> I have found that the easiest step to make someone more private is getting them to start using a private web browser. For someone willing to take a privacy and security quiz, they probably want to be more private and would be willing to download a private web browser, since it's easier than buying a shredder, safe or even configuring a router
> 
> Also Zone 3 has this question, and I believe the question in Zone 3 should be "Only use Tor for web browsing" since that's such an extreme threat model
> 
> I feel like this question should be moved, but if you disagree let me know

Your issue [49](https://github.com/techlore/website/issues/49):

> There's already a question about phone number aliasing in Zone 2, but a separate question should be added for email aliasing, since email aliasing is a lot easier than phone number aliasing
> 
> This question can even be put in zone 1 because it's a decently streamlined experience to create masked emails, and duckduckgo even allows unlimited masked emails for free, for anyone that doesn't want to pay for the other alternatives

Your issue [50](https://github.com/techlore/website/issues/50):

> This is a premature issue to be opened in this repository, since the feature hasn't released yet, but Brave's "forget this site" feature will make clearing cache and cookies so much easier and user friendly that I feel it has to be mentioned in Zone 1 next to "Clear device temporary data" and "frequently clear or never store browsing data"

**Comment by henry-fisher** [comment]
https://github.com/techlore/website/commit/526fe4b5aef3341ae0a6a7596d4d416d6ddefd3b
I went ahead and updated wording on biometrics in the newest changes to fix your concern @davidcollini 

I've got everything where I think is a good balance between technical accuracy while still being accessible to anyone. Keep in mind this quiz is meant to be taken by people who know nothing about privacy as well, so we're trying to keep it easy to follow. 

Loosely some upcoming projects I want to do video content on is accompanying videos that correspond with questions to offer more context for each question.
